package.json

```diff
@@ -31,25 +33,36 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "yarn stripes test karma"
+    "test:unit": "jest --ci --coverage && yarn run test:unit:report",
+    "test:unit:report": "cp -r ./artifacts/coverage-jest ./artifacts/coverage",
+    "test:e2e": "yarn stripes test karma --karma.singleRun --karma.browsers ChromeDocker",
+    "test": "yarn run test:unit && yarn run test:e2e"
   },
   "devDependencies": {
+    "@babel/plugin-transform-runtime": "^7.13.9",
     "@bigtest/cli": "^0.2.2",
     "@bigtest/interactor": "^0.9.2",
     "@bigtest/mocha": "^0.5.2",
     "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^5.0.0",
     "@folio/stripes": "^6.0.0",
     "@folio/stripes-cli": "^2.0.0",
-    "@testing-library/react": "^11.0.2",
-    "babel-eslint": "^10.0.0",
+    "@testing-library/dom": "^7.29.6",
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.5",
+    "@testing-library/user-event": "^12.8.1",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^26.6.3",
     "chai": "^4.2.0",
     "core-js": "^3.6.1",
     "eslint": "^7.9.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.0.1",
     "faker": "^5.1.0",
+    "jest": "^26.6.3",
+    "jest-css-modules": "^2.1.0",
```

Comment:
I'm not sure we use this

Reply:
this library is simply implementing identity-obj-proxy

Reply:
and you could point where it's used? I don't see any

Reply:
so the first moment that this library is not used anywhere, second - author recommends to look for jest documentation and use identity-obj-proxy directly. https://www.npmjs.com/package/jest-css-modules

> The Jest documentation provides more details on using identity-obj-proxy to solve this problem directly, and I recommend that you use that library instead of this one.

Reply:
https://github.com/folio-org/ui-plugin-find-po-line/pull/80